### PR TITLE
10K doodle

### PR DESCRIPTION
**Pull request by madhead**
10K doodle for #607.

### CODE REVIEW

File: app-frontend/app/src/components/head/logos.ts [hunk 1]
```diff
@@ -69,5 +69,18 @@ export const LOGOS: Logo[] = [
...
+  },
+  {
+    src: require("./10K.svg"),
+    alt: () => {
+      return `Celebrating 10 Years of Kotlin!`
+    },
+    show: () => {
+      const today = new Date();
+
+      return today.getMonth() === 6 && today.getDate() === 19;
```
Comment (IRus) [inline app-frontend/app/src/components/head/logos.ts:81]: What about date range? I want to more people enjoy this doodle :)

**Review by IRus — APPROVED** [review]

File: app-frontend/app/src/components/head/logos.ts (hunk 1 above)
Comment (madhead) [reply app-frontend/app/src/components/head/logos.ts]: Ok, made a range: minus week before the date and two days after.

**Comment by IRus** [comment]
image
Looks great!

**Comment by madhead** [comment]
@IRus, what is this extension to test times in a browser?!

**Comment by IRus** [comment]
I just update system settings :)

**Pull request merged**